mclagsyncd/mclaglink.cpp

```diff
@@ -687,46 +1859,62 @@ uint64_t MclagLink::readData()
         {
             case MCLAG_MSG_TYPE_PORT_ISOLATE:
                 setPortIsolate(msg);
-                break;
+                    break;
 
             case MCLAG_MSG_TYPE_PORT_MAC_LEARN_MODE:
                 setPortMacLearnMode(msg);
-                break;
+                    break;
 
             case MCLAG_MSG_TYPE_FLUSH_FDB:
                 setFdbFlush();
-                break;
+                    break;
 
             case MCLAG_MSG_TYPE_FLUSH_FDB_BY_PORT:
                 setFdbFlushByPort(msg);
-                break;
+                    break;
 
             case MCLAG_MSG_TYPE_SET_INTF_MAC:
                 setIntfMac(msg);
-                break;
+                    break;
 
             case MCLAG_MSG_TYPE_SET_FDB:
                 setFdbEntry(msg, (int)(hdr->msg_len - sizeof(mclag_msg_hdr_t)));
-                break;
-
-            case MCLAG_MSG_TYPE_GET_FDB_CHANGES:
-                write = getFdbChange(m_messageBuffer_send);
-                if (write == 0)
-                    throw MclagConnectionClosedException();
-                if (write < 0)
-                    throw system_error(errno, system_category());
-                break;
-
+                    break;
+            case MCLAG_MSG_TYPE_SET_TRAFFIC_DIST_ENABLE:
+            case MCLAG_MSG_TYPE_SET_TRAFFIC_DIST_DISABLE:
+                    mclagsyncd_set_traffic_disable(msg, hdr->msg_type);
+                    break;
+            case MCLAG_MSG_TYPE_SET_ICCP_STATE:
+                    mclagsyncd_set_iccp_state(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_SET_ICCP_ROLE:
+                    mclagsyncd_set_iccp_role(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_SET_ICCP_SYSTEM_ID:
+                    mclagsyncd_set_system_id(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_DEL_ICCP_INFO:
+                    mclagsyncd_del_iccp_info(msg);
+                    break;
+            case MCLAG_MSG_TYPE_SET_REMOTE_IF_STATE:
+                    mclagsyncd_set_remote_if_state(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_DEL_REMOTE_IF_INFO:
+                    mclagsyncd_del_remote_if_info(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_SET_PEER_LINK_ISOLATION:
+                    mclagsyncd_set_peer_link_isolation(msg, mclag_msg_data_len(hdr));
+                    break;
+            case MCLAG_MSG_TYPE_SET_ICCP_PEER_SYSTEM_ID:
+                    mclagsyncd_set_peer_system_id(msg, mclag_msg_data_len(hdr));
+                    break;
             default:
-                break;
+                    break;
```

Comment:
Please fix alignment

Reply:
@vsenchyshyn  thanks for comments. will address it